packages/backend/src/services/ProjectService/ProjectService.test.ts

```diff
@@ -129,4 +140,59 @@ describe('ProjectService', () => {
             expect(result).toEqual(expectedExploreSummaryFilteredByName);
         });
     });
+    describe('getJobStatus', () => {
+        test('should get job with projectUuid if user belongs to org ', async () => {
```

Comment:
Is there extra logic that these test that isn't captured in the `Ability` tests?

Reply:
`should not see error message if only has view permissions` is more specific, the others are a bit redundant, but it helps to test the integration with permissions... now that they are implemented I don't see any reason to remove them, the more tests the better